In [4]:
count=0
def count_digit(num):
    global count
    if num == 0:
        return 1
    if (num >0):
        if(num%10==0):
            count +=1
        count_digit(num // 10)
    return count
n=int(input())
print(count_digit(n))

00010204
2


Note: Even though "00010204" has 5 zeros, the output would still be 2 because when you convert it to an integer, it becomes 10204.